--- java/core/BUILD	2021-08-19 18:35:07.000000000 +0200
+++ java/core/BUILD	2021-08-19 18:35:07.000000000 +0200
@@ -126,17 +126,10 @@
         [
             "src/main/java/com/google/protobuf/*.java",
         ],
-        exclude = LITE_SRCS,
     ) + [
         "//:gen_well_known_protos_java",
     ],
     visibility = ["//visibility:public"],
-    exports = [
-        ":lite_runtime_only",
-    ],
-    deps = [
-        ":lite_runtime_only",
-    ],
     resources = [
         "//:well_known_protos",
     ],
